packages/code-snippet/src/CodeSnippetWidget.tsx

```diff
@@ -178,6 +179,15 @@ class CodeSnippetDisplay extends MetadataDisplay<
     return (widget as DocumentWidget).content instanceof FileEditor;
   };
 
+  // Return the language of the editor or empty string
+  private getEditorLanguage = (
+    widget: DocumentWidget<FileEditor>
+  ): string | undefined => {
+    const editorLanguage =
+      widget.context.sessionContext.kernelPreference.language;
+    return editorLanguage === 'null' ? '' : editorLanguage;
```

Comment:
is there a reason you're checking for the string containing null here now instead of checking for undefined?

Reply:
Yes. Plain text editors will have this field set to null as a string.